File: Scripts/newTask/file-input-output/javascript/file-input-output.js

for ( var i = 0; i < 1000000; i++) {
fs = require('fs');
fs.readFile('../input.txt', 'utf8', function (err, data) {
            if (err) throw err;
            //Do your processing, MD5, send a satellite to the moon, etc.
            fs.writeFile ('output.txt', data, function(err) {
                if (err) throw err;
            });
        });
}

File: Scripts/newTask/file-input-output/go/file-input-output.go
package main

import (
    "fmt"
    "io/ioutil"
)

func main() {

for i := 0; i < 1000000; i++ {

    b, err := ioutil.ReadFile("../input.txt")
    if err != nil {
        fmt.Println(err)
        return
    }
    if err = ioutil.WriteFile("output.txt", b, 0666); err != nil {
        fmt.Println(err)
    }
}
}
